src/legacy/server/kbn_server.js

```diff
@@ -36,7 +36,7 @@ import configCompleteMixin from './config/complete';
 import optimizeMixin from '../../optimize';
 import * as Plugins from './plugins';
 import { indexPatternsMixin } from './index_patterns';
-import { savedObjectsMixin } from './saved_objects';
+import { savedObjectsMixin } from './saved_objects/saved_objects_mixin';
```

Comment:
Importing directly from the `saved_objects_mixin` to avoid having to add types.

Reply:
`savedObjectsMixin` used to be exported by `./saved_objects/index` which is now a TS file. We will probably delete the mixin once the migration to the new platform is complete, so it also feels like we don't want to export this from `./saved_objects/index` even if we could add a ts-ignore comment.